### ISSUE TITLE
Anti-osteoprotegerin antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10287]
**Preferred term label:**
Anti-osteoprotegerin antibody positivity

**Synonyms**
Anti-OPG antibody positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies (immunoglobulins) in the blood circulation that react against osteoprotegerin.
PMID: 19812402
PMID: 28534161
PMID: 26092508

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Celiac disease and severe osteoporosis and high bone turnover.

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
added as
Adding new term: 
- ID: HP:6000193
- label: Anti-osteoprotegerin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:19812402|PMID:28534161|PMID:26092508
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against osteoprotegerin.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10287
- synonyms:

**Comment by pnrobinson** [comment]
reopening -- yes, let's add that as a synonym

**Comment by pnrobinson** [comment]
added synonym.
